test/atomic/test_wait64_notify.c

```diff
@@ -91,6 +96,7 @@ int main() {
 #ifdef __EMSCRIPTEN_WASM_WORKERS__
   emscripten_wasm_worker_t worker = emscripten_create_wasm_worker(stack, sizeof(stack));
   emscripten_wasm_worker_post_function_v(worker, worker_main);
+  emscripten_runtime_keepalive_push();
```

Comment:
I slightly worry about converting all the tests to this form, as our users may be writing in the simpler way?

Reply:
I guess I can make this part implicit by having `post_function` itself keep the runtime alive.

The other part of replacing `REPORT_RESULT` with normal exit I hope/think is fine.

Reply:
Actually post_function doesn't track when job is done so that don't work.

I think the benefit here of making these test runnable under node, along with using a more normal exit-status-based test setup outweighs the downside here of having code that a user might not write.   Note that `emscripten_runtime_keepalive_push` simply does nothing in default build without `-sEXIT_RUNTIME` since the runtime always stays alive regardless in the default mode.

Reply:
OK, maybe we can land just a small subset for now? #22777

Reply:
I'd be ok landing the whole thing, so long as we keep one test in the old form (something like a "wasm workers without exit runtime" test).